src/extensions/yfm/YfmDist/index.ts

```diff
@@ -17,7 +17,7 @@ export const YfmDist: ExtensionAuto = (builder) => {
             new Plugin({
                 props: {
                     attributes: {
-                        class: 'yfm yfm_no-list-reset',
+                        class: 'yfm yfm_no-list-reset yfm-editor',
```

Comment:
why here yfm-editor?

Reply:
for backward compatibility, as some services use this class name to override styles in markdown content in editor